src/fast-websocket.lisp

```diff
@@ -60,11 +60,12 @@
                (type integer start end))
       (ecase (opcode-name (ws-opcode ws))
         (:continuation
-         (fast-write-sequence payload buffer start end)
+         (let ((payload (subseq payload start end)))
```

Comment:
As `subseq` allocates a new array, it may degrade at the speed.
It's better to change `mask-message` to take `start` & `end`, then, no memory copying is needed.

Reply:
I think that was what was broken to start with using fast-write-sequence before, as this is not my area of expertise, I humbly ask if you can make the change instead, I do not wish to make a mistake in your work.

Reply:
Okay, I'll try it when I have time.
For now, I'm going to merge this :)
Thanks!